flask_dynamo/manager.py

```diff
@@ -148,6 +149,40 @@ def _get_ctx(app):
                 'flask-dynamo extension not registered on flask app'
             )
 
+    @staticmethod
+    def _init_session(app):
+        session_kwargs = {}
+        # Only apply if manually specified: otherwise, we'll let boto
+        # figure it out (boto will sniff for ec2 instance profile
+        # credentials).
+        if app.config['AWS_ACCESS_KEY_ID']:
```

Comment:
I think this is fine. I like the pattern of 'load at startup' and figure out all missing stuff then. Then we can rely on things being set for the duration of the program.